external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -284,7 +284,7 @@ def slice(self):
 
 @dataclasses.dataclass(frozen=True)
 class PackerConfig:
-    clip: ClipConfig
+    clip: ClipConfig = dataclasses.field(default_factory=dict)
```

Comment:
Shouldn't the default factory be `ClipConfig`?

Reply:
`ClipConfig` is just defined as `Mapping[Hashable, Mapping[str, SliceConfig]]`, is it possible to use that as a default factory?

Reply:
I added a factory function for ClipConfig, LMK if that's not what you meant.